### DISCUSSION TITLE
Does Geist have a Figma library for mockups?

### DISCUSSION THREAD

**Discussion by Zk2u** [zeit-ui/react#528] [Q&A] [answered]
self-explanatory

**Comment by unix** [comment, marked as answer]
Not yet.

**Reply by Zk2u** [reply]
by 'yet', i assume it's planned?

**Reply by unix** [reply]
😿 No such plan.

**Reply by Zk2u** [reply]
> 😿 No such plan.

we (@syvita) have plans to use Geist, should we create one internally that may be modded for ourselves, then create one without the modifications to this repo so others can use it?

**Reply by unix** [reply]
Do you mean to create a Figma project, or fork Geist and modify it?  (I don't understand what you mean)
(We don't have Figma library because we don't have the manpower to maintain more projects at this stage)

**Reply by Zk2u** [reply]
Both. Our design team will probs add various things to it when doing our mockups for projects, but we'll most likely create all the components from here in Figma. We'll also end up making these changes in a react component lib fork of this so we can use them. But since we're putting most of the components into our fork anyway, we could work to release a non-modified version. Manpower shouldn't be much of an issue for us and I think we'd love to contribute

**Reply by unix** [reply]
All contributions are welcome, and Geist is happy to accept features if they are intended for the general public, if it is a domain-specific function or strongly business-related, it may not be merged. 
For the sake of your scalability, I suggest you fork a copy for development and then merge any features you see fit back in here.

**Comment by Mootbing** [comment]
Reopening this
